### ISSUE TITLE
No data is stored

### ISSUE THREAD

**Issue by RuralHunter** [GChristensen/scrapyard#53]
Recently I found when I try to open the imported scrapbook items, scrapyard reports a "No data is stored" error. It worked without problem before. What happened?

**Comment by RuralHunter** [comment]
I tried to re-import the scrapbook rdf. But the same error occurs with the new imported items.

**Comment by GChristensen** [comment]
Most likely the saved pages are missing from disk or became inaccessible somehow. Please check the presence/accessibility of the missing items (this may be tricky). If import works smoothly (it opens and closes tabs) and all imported pages are missing, I can not say anything without seeing debug log.

**Comment by RuralHunter** [comment]
How to get the debug log? I can see all the data files in Scrapbook folder. Does Scrapyard copy the data files from Scrapbook when doing the import? if yes where are they stored? It's not only some items missing. Actually none is working.

**Comment by GChristensen** [comment]
>How to get the debug log?

It is under the "Console" tab at the following url: **about:devtools-toolbox?id=scrapyard-we%40firefox&type=extension**
Please try to look at it when importing. If there are no errors, and no tabs are opened (assuming that the "Quick import" checkbox is unchecked), I probably will not be able to help and this must be some local file-system related issue.

>I can see all the data files in Scrapbook folder. Does Scrapyard copy the data files from Scrapbook when doing the import? if yes where are they stored?

Under <scrapbook root>/data folder there should be date-labeled folders, under which files named **index.html** are located along with other media files. index.html files are loaded and saved during the import.

**Comment by RuralHunter** [comment]
I saw only one error when doing the import. I translated the localized error message like this:
Security error：The content located at moz-extension://ec27493e-19fb-4915-bf97-b8e3d020070c/ui/sidebar.html can not be loaded or linked to file:///D:/AppProfile/Firefox/ScrapBook/data/20060228105149/favicon.ico.

The scrapbook path reported in the message is not correct. My firefox profile location is actually D:/Firefox, not D:/AppProfile/Firefox. Where does the wrong path come from?

**Comment by GChristensen** [comment]
It seems that this is a filesystem issue and there is nothing wrong with Scrapyard. This only may come from "Path to the RDF file" textbox at the RDF import tab. You need to specify a proper path to your scrabook.rdf.

**Comment by RuralHunter** [comment]
hmm....I definitely inputed the correct path in the RDF import tab. There are lots of successful http access logs in the console with just only one error.

**Comment by GChristensen** [comment]
There are many unknowns, so I could not say anything. You may try to:
1. Use a quick import. In this case pages would not be imported into Scrapyard but served from disk. See if this works.
2. Copy scrapbook folder to  D:/AppProfile/Firefox/ScrapBook, or create a [junction](https://docs.microsoft.com/en-us/sysinternals/downloads/junction) with this name and try to import from there. 
3. Search "AppProfile" in scrapbook.rdf file. If it is hardcoded there, may be Scrapyard does not knows about some ScrapBook RDF peculiarities.

**Comment by RuralHunter** [comment]
All the operations I tried above are "quick import". You reminder me and I I tried a full import(unchecked the quick import check box). Now it works. So there is something wrong with quick import only. 
I searched "AppProfile" in the rdf file and did find some occurrences but it seems they are only icon paths and nothing really important:
```
NS2:icon="file:///D:/AppProfile/Firefox/ScrapBook/data/20051218112106/favicon.ico"
```
Anyway, the full import has no problem. Thanks for your help.
